Source/qol/common.cpp

```diff
@@ -25,16 +25,12 @@ int GetTextWidth(const char *s)
 
 void FastDrawHorizLine(const CelOutputBuffer &out, int x, int y, int width, Uint8 col)
 {
-	memset(out.at(x, y), col, width);
+	UnsafeDrawHorizontalLine(out, {x, y}, width, col);
 }
 
 void FastDrawVertLine(const CelOutputBuffer &out, int x, int y, int height, Uint8 col)
 {
-	BYTE *p = out.at(x, y);
-	for (int j = 0; j < height; j++) {
-		*p = col;
-		p += out.pitch();
-	}
+	UnsafeDrawVerticalLine(out, {x, y}, height, col);

```

Comment:
I think you can kill that and just directly use the UnsafeDraw* functions.

Reply:
Yeah I'll do it at the very end, don't want to have conflicts in case QoL stuff gets changed in the meantime